frontend/src/pages/Editor.jsx

```diff
@@ -31,7 +32,7 @@ const Editor = () => {
 
   // canvas pixel data
   const [canvasData, setCanvasData] = useLocalStorage("canvas", defaultCanvas);
-
+  const [eyeDropperColor, setEyeDropperColor] = useLocalStorage("eyeDropperColor", "#FFFFFF");
```

Comment:
eye dropper color does not need to be local storage. We can change this to useState or even just simply have the value from the callback set to primary directly.